pkg/cardtoken/client.go

```diff
@@ -0,0 +1,45 @@
+package cardtoken
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/httpclient"
+)
+
+const (
+	baseURL = "https://api.mercadopago.com/v1/"
+	urlGet  = baseURL + "/v1/card_tokens/{id}"
+	urlPost = baseURL + "/v1/card_tokens"
+)
+
+type Client interface {
```

Comment:
Para seguir o padrão dos demais clients
```suggestion
// Client contains the method to interact with the card token API.
type Client interface {
```

Reply:
Ajustado!!